### PR TITLE
Saint-Roumain Militia 

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Implements the [Saint-Roumain Militia](https://hackmd.io/ZHF_aRWsQ7OKNBT0DUuE-A), the hunters of Illestren.

image

Redoes the Riggs C, however doesn't enable it quite yet. It should be introduced via admin event.

This also adds a new loadout outfit that makes you feel like you ~~are stuck at home~~ need to quest

This also Fixes wood flooring colors
image
A unintended consequence of this is that ships that were made with the dark wood flooring now look weird. This can be fixed for now by using the tiles in hand and swapping them out for the different wood colors

## Why It's Good For The Game

The more lore in game, the better.

The 7 unused wood colors because of the baytile port forgetting to add them fully was also making me sad.

## Changelog
:cl:
add: New minor faction: Saint-Roumain Militia
add: A new loadout outfit that makes you feel like you need to quest
add: The Rigoureux has gotten a makeover, courtesy of the Roumains. It is still at drydock.
del: The old Rigoureux  has been renamed to Riggs-T and is unbuyable
fix: fixes wood tile colors (ships that look odd with the fixed colors can use the floors tiles in hand to recolor them)
imageadd: floor tile object resprite
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -333,3 +333,24 @@
 	icon_state = "inteq_battlecoat"
 	item_state = "inteq_battlecoat"
 	supports_variations = DIGITIGRADE_VARIATION
+
+/obj/item/clothing/suit/armor/roumain
+	name = "saint-roumain duster"
+	desc = "A coat made from hard lather. Meant to withstand long durations in the wild."
```
Comment (Apogee-dev) [inline code/modules/clothing/suits/armor.dm]: desc = "A coat made from hard leather. Meant to withstand long hunts in harsh wilderness."

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -333,3 +333,24 @@
...
+	icon_state = "rouma_coat"
+	item_state = "rouma_coat"
+	body_parts_covered = CHEST|GROIN|ARMS
+	cold_protection = CHEST|GROIN|ARMS
+	heat_protection = CHEST|GROIN|ARMS
+
+/obj/item/clothing/suit/armor/roumain/shadow
+	name = "saint-roumain shadow duster"
+	desc = "A coat made from hard lather. To seperate the trainees from everyone else."
```
Comment (Apogee-dev) [inline code/modules/clothing/suits/armor.dm]: desc = "A coat made from hard leather. Its rough, barely-treated finish is typical of one of the Saint-Roumain Militia's trainees."

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -333,3 +333,24 @@
...
+	icon_state = "rouma_shadow_coat"
+	item_state = "rouma_shadow_coat"
+
+/obj/item/clothing/suit/armor/hos/roumain/montagne
+	name = "saint-roumain montagne coat"
+	desc = "A stylish red coat to signal that you are, in fact, a montagne. Made of extra hard leather, infused with bulletproof material. Somehow."
```
Comment (Apogee-dev) [inline code/modules/clothing/suits/armor.dm]: desc = "A stylish red coat to indicate that you are, in fact, a Hunter Montagne. Made of extra hard exotic leather, treated with bullet-resistant materials, and lined with the fur of some unidentifiable creature."

File: code/modules/clothing/suits/labcoat.dm [hunk 4]
```diff
@@ -64,6 +64,12 @@ You're pretty sure this is just a raincoat.
 	mob_overlay_icon = 'icons/mob/clothing/suit.dmi' // same as above
 	item_state = "raincoat"
 
+/obj/item/clothing/suit/toggle/labcoat/roumain_med
+	name = "saint-roumain medical duster"
+	desc = "A coat made from hard lather infused with labcoat material. Due to the infusion, it is better at resisting biological hazards."
```
Comment (Apogee-dev) [inline code/modules/clothing/suits/labcoat.dm]: desc = "A coat made from hard leather and further treated with exotic sterilizing oils and wax. The treatment and its more closed design offers much better protection against biological hazards."

File: code/modules/clothing/under/suits.dm [hunk 5]
```diff
@@ -138,3 +138,23 @@
 	item_state = "henchmen"
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS|HEAD
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEEARS|HIDEEYES|HIDEHAIR
+
+/obj/item/clothing/under/suit/roumain
+	name = "saint-roumain's worksuit"
+	desc = "An suit designed for the hardworking hunters."
```
Comment (Apogee-dev) [inline code/modules/clothing/under/suits.dm]: desc = "A simple, hard-wearing suit designed for the hardworking hunters of the Saint-Roumain Militia."

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 6]
```diff
@@ -270,3 +270,11 @@
 		user.emote("scream")
 		user.drop_all_held_items()
 		user.Paralyze(80)
+
+/obj/item/gun/ballistic/revolver/pepperbox
+	name = "\improper pepperbox pistol"
+	desc = "A extremly old revolver, how it even end up here? While fast to fire, it is extremely innacurate. Uses .357 ammo."
```
Comment (Apogee-dev) [inline code/modules/projectiles/guns/ballistic/revolver.dm]: desc = "An archaic precursor to revolver-type firearms, this gun was rendered completely obsolete millennia ago. How did it even end up here? While fast to fire, it is extremely inaccurate. Uses .357 ammo."

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 7]
```diff
@@ -385,15 +385,26 @@
...
 	inhand_y_dimension = 32
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/winchester
 	fire_sound = 'sound/weapons/gun/rifle/shot.ogg'
+
+/obj/item/gun/ballistic/shotgun/winchester/lethal
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/winchester/lethal
+
+/obj/item/gun/ballistic/shotgun/winchester/mk1
+	name = "Winchester MK.1"
+	desc = "A sturdy lever action rifle. This one is an antique and old model."
```
Comment (Apogee-dev) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]: desc = "A sturdy lever action rifle. This older pattern appears to be an antique, in excellent condition despite its age."

File: whitesands/code/modules/clothing/head/jobs.dm [hunk 8]
```diff
@@ -46,3 +46,23 @@
 	icon_state = "cowboyhos"
 
 	dog_fashion = /datum/dog_fashion/head/cowboy
+
+/obj/item/clothing/head/cowboy/sec/roumain
+	name = "hunter's hat"
+	desc = "A fancy hat with a nice feather. It covers your eyes, which makes you feel badass."
```
Comment (Apogee-dev) [inline whitesands/code/modules/clothing/head/jobs.dm]: desc = "A fancy hat with a nice feather. The way it covers your eyes makes you feel like a badass."

File: whitesands/code/modules/clothing/head/jobs.dm [hunk 9]
```diff
@@ -46,3 +46,23 @@
...
+	icon_state = "rouma_hat"
+
+/obj/item/clothing/head/cowboy/sec/roumain/shadow
+	name = "shadow's hat"
+	desc = "A okay looking hat. It covers your eyes, which makes you feel badass, but in reality you look like a wannabe hunter."
```
Comment (Apogee-dev) [inline whitesands/code/modules/clothing/head/jobs.dm]: desc = "A rough, simple hat. The way it covers your eyes makes you feel badass, but you just look like a wannabe hunter."

File: whitesands/code/modules/clothing/head/jobs.dm [hunk 10]
```diff
@@ -46,3 +46,23 @@
...
+	icon_state = "rouma_shadow_hat"
+
+/obj/item/clothing/head/cowboy/sec/roumain/med
+	name = "medical hunter's hat"
+	desc = "A hat that makes you want to cure plauges."
```
Comment (Apogee-dev) [inline whitesands/code/modules/clothing/head/jobs.dm]: desc = "A very wide-brimmed, round hat treated with oil and wax. Somehow manages to look stylish and creepy at the same time."

File: whitesands/code/modules/clothing/head/jobs.dm [hunk 11]
```diff
@@ -46,3 +46,23 @@
...
+	icon_state = "rouma_med_hat"
+
+/obj/item/clothing/head/cowboy/sec/roumain/montagne
+	name = "montagne's hat"
+	desc = "A very fancy hat to signal that you are, in fact, a montagne."
```
Comment (Apogee-dev) [inline whitesands/code/modules/clothing/head/jobs.dm]: desc = "A very fancy hat with a large feather plume to signify that you are, in fact, a Hunter Montagne. The exotic fur lining is impeccably soft and bafflingly bulletproof."

I'd also suggest up-armoring this one to match the other HoS headgear or re-pathing it to /obj/item/clothing/head/HoS/cowboy/montagne like i did with the sheriff cowboy hat.

**Comment by Valorium-RB** [comment]
Looks lovely! All the descriptions sound great, too.

File: _maps/configs/srm_rigger.json [hunk 12]
```diff
@@ -0,0 +1,25 @@
+{
+	"map_name": "Saint-Roumain Modified Rigoureux-class Auxiliary Cutter",
+	"map_short_name": "Rigoureux-class",
```
Comment (Apogee-dev) [inline _maps/configs/srm_rigger.json:5]: You should add lines for prefix and ship namelists in this section.

Comment (rye-rice) [reply _maps/configs/srm_rigger.json:5]: SRV?

Comment (Apogee-dev) [reply _maps/configs/srm_rigger.json:5]: SRV or SRSV would work well for prefix, yeah. Take your pick for namelists, though we don't have a suitably religious one. Maybe use the mythological and beasts lists.

Comment (Valorium-RB) [reply _maps/configs/srm_rigger.json:5]: Mythological and beasts works. Nature or firearm themes work too, if you wanna add new ones.

**Comment by rye-rice** [comment]
conflicts fixed

**Comment by rye-rice** [comment]
conflicts fixed

**Comment by rye-rice** [comment]
saint-conflict militia 

fixed conflicts

**Comment by rye-rice** [comment]
ocnflucts fixed

**Comment by PrefabQuasar** [comment]
could we get a picture of the remapped Rigoureux?

**Comment by rye-rice** [comment]
> could we get a picture of the remapped Rigoureux?

image

**Comment by rye-rice** [comment]
updated the map. at this point jusut merge it, the revamped ship isnt even spawnable, and we could just make it spawnable later

**Comment by rye-rice** [comment]
vhyuig y76trde567e4rtdst5674erdsx54esr5ye4rs5t4r3wesast

**Comment by ZephyrTFA** [comment]
> vhyuig y76trde567e4rtdst5674erdsx54esr5ye4rs5t4r3wesast

Do I need to call a wambulance

**Comment by triplezeta** [comment]
to hell with it, let's get this merged

**Pull request merged**